components/epaxos/src/instance/instance.rs

```diff
@@ -21,12 +21,12 @@ pub enum InstanceStatus {
     Executed,
 }
 
-pub struct DepCmds(Vec<InstanceID>);
+pub struct InstIDs(Vec<InstanceID>);
 
 pub struct Instance {
     pub status: InstanceStatus,
-    pub cmds: Vec<command::Command>,
-    pub ballot: replica::BallotNum,
+    pub cmds: Vec<Command>,
+    pub ballot: BallotNum,
     pub seq: u64,
```

Comment:
pub type Sequence = i64;
```suggestion
    pub seq: Sequence,
```
下面有些定义seq i32, i64

Reply:
也行，所有的seq都用u64类型吧

Reply:
i64吧...无符号的我心里不踏实